unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -61,7 +62,12 @@ public void SetCatalog(WearableDictionary catalog)
 
     public void LoadUserProfile(UserProfile userProfile)
     {
-        if (userProfile?.avatar == null || string.IsNullOrEmpty(userProfile.avatar.bodyShape)) return;
+        LoadUserProfile(userProfile, false);
+    }
+
+    public void LoadUserProfile(UserProfile userProfile, bool forceLoading)
+    {
+        if ((!forceLoading && renderingEnabled && !view.isOpen) || userProfile?.avatar == null || string.IsNullOrEmpty(userProfile.avatar.bodyShape)) return;
```

Comment:
Maybe we can use `Application.isBatchMode` here to skip that conditional, so the tests don't break?